src/components/dashboard/sidebar/SidebarCard.jsx

```diff
@@ -10,17 +10,17 @@ const SidebarCard = ({
 }) => (
   <Card className={cardClassNames}>
     {title && <Card.Header title={title} />}
-    <Card.Section
+    {children && <Card.Section
       className={cardSectionClassNames}
     >
       {children}
-    </Card.Section>
+    </Card.Section>}
```

Comment:
This conditional was added to disable rendering a card section when no children are passed. Not adding this conditional would add the default styling associated with a Card section (specifically the padding)